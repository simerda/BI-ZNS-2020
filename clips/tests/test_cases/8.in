ano
6
6
ano
ano
ano
ne
