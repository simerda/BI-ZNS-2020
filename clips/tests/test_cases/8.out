CLIPS> Mas figurku ve hre (ano/ne)?
Co jsi naposled hodil (1/2/3/4/5/6)?
Hazej znovu, kolik padlo (1/2/3/4/5/6)?
Chces nasadit (mas-li k dispozici) novou figurku (ano/ne)?
Je na tvem domecku volno/cizi figurka (ano/ne)?
Nasad novou figurku.
Mas nejakou figurku, ktere zbyva N poli do domecku? (ano / ne)
Mas nejakou figurku, ktera ma N poli pred sebou cizi figurku? (ano / ne)
Tahni tou figurkou.

